clients/assessment_component/src/assessment/pages/components/diagrams/BarChartWithScoreLevel.tsx

```
@@ -3,7 +3,9 @@ import { BarChart, Bar, LabelList, XAxis, YAxis, Rectangle } from 'recharts'
 import { useMemo } from 'react'
 import { chartConfig } from './utils/chartConfig'
 import { getCornerRadius } from './utils/getCornerRadius'
-import { DataPoint } from '../interfaces/DataPoint'
+import { DataPoint } from './interfaces/DataPoint'
+
+import { ScoreLevel } from '../../../interfaces/scoreLevel'
 
 import { ScoreLevel } from '../../../interfaces/scoreLevel'
```

Biome (1.9.4)

[error] Shouldn't redeclare 'ScoreLevel'. Consider to delete it or rename it.

'ScoreLevel' is defined here:


(lint/suspicious/noRedeclare)